gapis/replay/replay_connection.go

```diff
@@ -108,6 +108,19 @@ func (e *backgroundConnection) HandlePayloadRequest(ctx context.Context, payload
 	return log.Errf(ctx, err, "Payload type is unexpected: %T", boxed)
 }
 
+// HandleProfilerReadyRequest implements gapir.ReplayResponseHandler interface.
+func (e *backgroundConnection) HandleProfilerReadyRequest(ctx context.Context, req *gapir.ProfilerReadyRequest, conn gapir.Connection) error {
+	ctx = status.Start(ctx, "Profile Request")
+	defer status.Finish(ctx)
+	waitId := req.GetId()
+	if waitId == 0 {
+		return log.Err(ctx, nil, "Invalid wait ID")
+	}
+	// TODO(apbodnar) Start a perfetto trace here
```

Comment:
This function should just forward it to the e.executor like the notifications above, that way the transform can handle this. It's OK to leave it as a TODO, but maybe mention that in this TODO.